lib/nimbusRecipe.ts

```diff
@@ -181,10 +181,22 @@ this._rawRecipe.localizations?.[Object.keys(this._rawRecipe.localizations)[0]])
         branchInfo.id = feature.value.messages[0].id
         break
     }
-
-    branchInfo.ctrDashboardLink =
-      getDashboard(branch.template, branchInfo.id, undefined, branchInfo.nimbusExperiment.slug, branch.slug)
-
+    // We are using the proposed end date + 1 as the end date because the end
+    // date is not inclusive in Looker
+    branchInfo.ctrDashboardLink = getDashboard(
+      branch.template,
+      branchInfo.id,
+      undefined,
+      branchInfo.nimbusExperiment.slug,
+      branch.slug,
+      branchInfo.nimbusExperiment.startDate,
+      getProposedEndDate(
+        branchInfo.nimbusExperiment.startDate,
+        branchInfo.nimbusExperiment.proposedDuration
+          ? branchInfo.nimbusExperiment.proposedDuration + 1
+          : undefined
```

Comment:
To make things more readable, please hoist the `getProposedEndDate` call into its own `const` above this statement so that the reader doesn't have to deal with multiple levels of abstraction as well as the ternary operator as part of understanding the main `getDashboard` invocation.